app/layout.tsx

```diff
@@ -121,7 +121,7 @@ export default async function RootLayout({
                   <SolanaProvider>
                     <SidebarProvider defaultOpen={defaultOpen}>
                       <TrackLastVisit />
-                      <WalletSidebarSync />
```

Comment:
Let's just remove this component. I don't like the idea of committing commented-out, dead code, the intent of this is confusing